pkg/audit/manager.go

```diff
@@ -1169,7 +1169,7 @@ func violationMsg(constraint *unstructured.Unstructured, enforcementAction util.
 		Kind:                  constraint.GetKind(),
 		Name:                  constraint.GetName(),
 		Namespace:             constraint.GetNamespace(),
-		EnforcementAction:     string(enforcementAction),
+		EnforcementAction:     enforcementAction,
```

Comment:
Backwards compatibility concern?

Reply:
To ensure backwards compatibility, we could add a new `EnforcementActions` instead and not touch the existing field. wdyt?